pkg/util/clusterdata/clusterdata.go

```diff
@@ -48,12 +51,15 @@ func NewBestEffortEnricher(log *logrus.Entry, env env.Interface) OpenShiftCluste
 type bestEffortEnricher struct {
 	log *logrus.Entry
 	env env.Interface
+	m   metrics.Interface
 
 	restConfig       func(env env.Interface, oc *api.OpenShiftCluster) (*rest.Config, error)
 	taskConstructors []enricherTaskConstructor
 }
 
 func (e *bestEffortEnricher) Enrich(ctx context.Context, ocs ...*api.OpenShiftCluster) {
+	e.m.EmitGauge("enricher.tasks.count", int64(len(e.taskConstructors)*len(ocs)), nil)
```

Comment:
Are these prometheus metrics?

I'm sure I'm coming in late and am going to mess everything up but naming convention is to use snake case (i.e. `enricher_tasks_count`)

Reply:
no - geneva